src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -6830,20 +6830,23 @@ namespace HVACUnitarySystem {
 						CalcUnitarySystemToLoad( UnitarySysNum, AirLoopNum, FirstHVACIteration, CoolPLR, HeatPLR, OnOffAirFlowRatio, TempSensOutput, TempLatOutput, HXUnitOn, _, _, CompressorONFlag );
 					} else if ( CoolingLoad ) {
 						// RegulaFalsi may not find cooling PLR when the latent degradation model is used.
-						// IF iteration limit is exceeded, find tighter boundary of solution and repeat RegulaFalsi
+						// IF iteration limit is exceeded (SolFlag = -1), find tighter boundary of solution and repeat RegulaFalsi
 						TempMaxPLR = -0.1;
 						TempSysOutput = SensOutputOff;
 						TempLoad = ZoneLoad;
-						while ( ( TempSysOutput - TempLoad ) > 0.0 && TempMaxPLR < 1.0 ) {
+						while ( ( TempSysOutput - TempLoad ) > 0.0 && TempMaxPLR < 0.95 ) { // avoid PLR > 1 by limiting TempMaxPLR to 1 (i.e., TempMaxPLR += 0.1)
```

Comment:
If RegulaFalsi hits max iteration limit finding a solution, this code tries to find a tighter bound on PLR. It steps in 0.1 increments, so 0.95 if the upper limit before PLR hits 1. If you recall, the UnitarySystem passed PLR=1.1 to the multispeed coil. In this loop, as 0.1 was added to PLR for the next iteration, as it approached 1, PLR was actually 0.999999, and the system did not exceed the load (which it should have but did not because of a bug) so this loop iterated once again to yield PLR=1.1. I never want that, or at least I don't think I do.